npm start does not work

I have this error there.

ERROR in ./src/example.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./containers/App.js in /Users/janmarek/www/react-intl-tel-input/src
 @ ./src/example.js 15:25-56

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -43,7 +43,7 @@ module.exports = {
       'styles': __dirname + '/src/styles',
       'mixins': __dirname + '/src/mixins',
       'components': __dirname + '/src/components/',
-      'react-intl-tel-input': './containers/App.js',
+      'react-intl-tel-input': './containers/IntlTelInputApp.js',
     },
     root: __dirname + '/src',
   },

